Makefile

```diff
@@ -1,5 +1,7 @@
 .DEFAULT_GOAL := help
 
+export GOEXPERIMENT=rangefunc
```

Comment:
let's add comment on top `todo remove it after update to go 1.23`

Reply:
[done](https://github.com/NethermindEth/juno/pull/1954/commits/3e10bbbb4d92e126b5da30f74ab415f41527032e)

Reply:
comments in Makefile starts with `#`